pkg/networkservice/common/timeout/server.go

```diff
@@ -97,3 +92,34 @@ func (s *timeoutServer) Close(ctx context.Context, conn *networkservice.Connecti
 	}
 	return &empty.Empty{}, err
 }
+
+func after(ctx context.Context, conn *networkservice.Connection) time.Duration {
+	clockTime := clock.FromContext(ctx)
+
+	var minTimeout *time.Duration
```

Comment:
```suggestion
	var minTimeout time.Duration = 1
```

Reply:
we can't set 1 here because we won't find a value less than that in Path segments